middleware/analytics/AnalyticsLoader.ts

```diff
@@ -0,0 +1,18 @@
+import { IAnalytics } from "./IAnalytics";
+import {consoleLogger} from "./console-logger";
+import {plausible} from "./plausible";
+
+
+export const AnalyticsLoader = function(ProviderName: string, environment:string) :IAnalytics {
+    if (environment==="test")
```

Comment:
Again if we can wrap this in a .env variable that would be ace.

Reply:
so - test is a value put into the environment by Jest - however I have also added an "Enabled" field that is read from the environment field also: 

If environment is test (you are running the tests) then don't load any analytics (as we don't want to spam the analytics when testing).
if enabled === false then don't load any analytics.